docker-compose.yml

```diff
@@ -23,17 +24,25 @@ services:
     restart: always
     depends_on:
       - "postgres"
-    container_name: group-ironmen-tracker-backend
-    ports:
-      - 5000:8080  # replace this if using a docker-compatible reverse proxy like traefik
+    container_name: backend
   postgres:
     image: postgres
     restart: always
-    container_name: group-ironmen-tracker-postgres
+    container_name: postgres
     environment:
       POSTGRES_USER: ${PG_USER}
       POSTGRES_PASSWORD: ${PG_PASSWORD}
       POSTGRES_DB: ${PG_DB}
     volumes:
       - ./pg-data:/var/lib/postgresql/data # change the left-hand side of : to the path you prefer the DB to store the data in
       - ./server/src/sql/schema.sql:/docker-entrypoint-initdb.d/schema.sql # change the left-hand side of the : to the path that contains the schema.sql
+  nginx:
+    image: nginx
+    depends_on:
+      - "backend"
+    ports:
+      - 3333:3333
+    volumes:
+      - ./nginx.conf:/etc/nginx/conf.d/default.conf
```

Comment:
This was my "solution" to the CORS issue here https://github.com/christoabrown/group-ironmen/issues/22 - I'm not convinced this is neccessary but 🤷 it got it working for me